terraform/SLI_3_days_views.tf

```diff
@@ -0,0 +1,83 @@
+resource "google_bigquery_table" "census_data_3_days_ago_view" {
+  project = "${var.slos_views_destination_project}"
+  dataset_id = "SLO_views_legacy"
```

Comment:
If you create dataset with TF, you could reference also dataset_id, so that it's configured in one place

Reply:
Done